apis/v1beta1/httproute_types.go

```diff
@@ -244,6 +244,55 @@ type HTTPRouteRule struct {
 	// +optional
 	// +kubebuilder:validation:MaxItems=16
 	BackendRefs []HTTPBackendRef `json:"backendRefs,omitempty"`
+
+	// Timeouts defines the timeouts that can be configured for an HTTP request.
+	//
+	// Support: Extended
+	//
+	// +optional
+	// <gateway:experimental>
+	Timeouts *HTTPRouteTimeouts `json:"timeouts,omitempty"`
```

Comment:
why use plural here

Reply:
I see your point, but I think it's the right name unless it is against Kubernetes naming conventions? This is just implementing GEP-1742, so if you think it requires more discussion you should probably bring it up in https://github.com/kubernetes-sigs/gateway-api/pull/2155

Reply:
can we resolve this @hzxuzhonghu @frankbu or does it need further discussion ?

Reply:
Yes, this can be resolved since this PR is just implementing GEP-1742. Any further discussion would need to be in an issue with GEP-1742.